### ISSUE TITLE
Fails when installed with pnpm

### ISSUE THREAD

**Issue by aleclarson** [babel/preset-modules#5]
When installed with [pnpm](https://pnpm.js.org), this library triggers an error in Babel core.

```
[!] (plugin babel) Error: [BABEL] ~/src/index.ts: Cannot find module '@babel/plugin-proposal-unicode-property-regex' from '~'
src/index.ts
Error: [BABEL] ~/src/index.ts: Cannot find module '@babel/plugin-proposal-unicode-property-regex' from '~'
    at Function.module.exports [as sync] (~/node_modules/.pnpm/registry.npmjs.org/resolve/1.12.0/node_modules/resolve/lib/sync.js:74:15)
    at resolveStandardizedName (~/node_modules/.pnpm/registry.npmjs.org/@babel/core/7.7.2/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePlugin (~/node_modules/.pnpm/registry.npmjs.org/@babel/core/7.7.2/node_modules/@babel/core/lib/config/files/plugins.js:54:10)
    at loadPlugin (~/node_modules/.pnpm/registry.npmjs.org/@babel/core/7.7.2/node_modules/@babel/core/lib/config/files/plugins.js:62:20)
    at createDescriptor (~/node_modules/.pnpm/registry.npmjs.org/@babel/core/7.7.2/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at ~/node_modules/.pnpm/registry.npmjs.org/@babel/core/7.7.2/node_modules/@babel/core/lib/config/config-descriptors.js:109:50
    at Array.map (<anonymous>)
    at createDescriptors (~/node_modules/.pnpm/registry.npmjs.org/@babel/core/7.7.2/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPluginDescriptors (~/node_modules/.pnpm/registry.npmjs.org/@babel/core/7.7.2/node_modules/@babel/core/lib/config/config-descriptors.js:105:10)
    at plugins (~/node_modules/.pnpm/registry.npmjs.org/@babel/core/7.7.2/node_modules/@babel/core/lib/config/config-descriptors.js:40:19)
```

**Note:** The `~` represents my library's root directory.

Advice for pnpm support can be found here:
https://pnpm.js.org/en/faq#pnpm-does-not-work-with-your-project-here

**Comment by nicolo-ribaudo** [comment]
They are already defined in `package.json` :thinking: 

https://github.com/babel/preset-modules/blob/2fd659008a71cbf6a368a55a50f61724eefbae86/package.json#L95-L98

**Comment by developit** [comment]
Fixed in #6.

**Comment by aleclarson** [comment]
@developit Can you release the fix please?
